### PR TITLE
RHDEVDOCS-4346 - Logging Corrections 

### PR DESCRIPTION
**Pull request by libander**
Fluentd to collector changes, minor formatting error corrections. 

Version(s): 4.8 +

Issue: [RHDEVDOCS-4346](https://issues.redhat.com//browse/RHDEVDOCS-4346), [RHDEVDOCS-3771](https://issues.redhat.com//browse/RHDEVDOCS-3771)

Link to docs preview: https://drive.google.com/file/d/1rZByPAvvYxnwdNZryCVoZ7bHRB3zkF5p/view?usp=sharing

Reviewers:
QE: @anpingli / @kabirbhartiRH  / @QiaolingTang 
SME: @alanconway / @jcantrill

### CODE REVIEW

File: modules/cluster-logging-elasticsearch-audit.adoc [hunk 1]
```diff
@@ -68,7 +68,7 @@ spec:
      secret:
        name: es-audit
    - name: secureforward-offcluster
-     type: "fluentdForward"
+     type: "collectorForward"
```
Comment (periklis) [inline modules/cluster-logging-elasticsearch-audit.adoc:71]: The type is still `fluentdForward`. Probably a greedy search/replace editor mistake here.

File: modules/cluster-logging-json-log-forwarding.adoc [hunk 2]
```diff
@@ -27,7 +27,7 @@ To enable parsing JSON log, you add `parse: json` to a pipeline in the `ClusterL
 ----
 pipelines:
 - inputRefs: [ application ]
-  outputRefs: myFluentd
+  outputRefs: mycollector
```
Comment (periklis) [inline modules/cluster-logging-json-log-forwarding.adoc:30]: I can't see the full CR here, but probably we want to keep `myFluentd`

File: modules/cluster-logging-troubleshooting-loki-entry-out-of-order-errors.adoc [hunk 3]
```diff
@@ -41,7 +41,7 @@ If your Fluentd forwards a large block of messages to a Loki logging system that
 ----
 ,\nentry with timestamp 2021-08-18 05:58:55.061936 +0000 UTC ignored, reason: 'entry out of order' for stream:
 
-{fluentd_thread=\"flush_thread_0\", log_type=\"audit\"},\nentry with timestamp 2021-08-18 06:01:18.290229 +0000 UTC ignored, reason: 'entry out of order' for stream: {fluentd_thread="flush_thread_0", log_type="audit"}
+{collector_thread=\"flush_thread_0\", log_type=\"audit\"},\nentry with timestamp 2021-08-18 06:01:18.290229 +0000 UTC ignored, reason: 'entry out of order' for stream: {collector_thread="flush_thread_0", log_type="audit"}
```
Comment (periklis) [inline modules/cluster-logging-troubleshooting-loki-entry-out-of-order-errors.adoc:44]: `fluentd_thread` is a label set by fluentd here, so you can't rename it to `collector_thread`

File: modules/cluster-logging-updating-logging-to-5-0.adoc [hunk 4]
```diff
@@ -167,14 +167,14 @@ green  open   audit-000001
...
+$ oc get ds collector -o json | grep collector-init
 ----
 +
-Verify that the output includes a `fluentd-init` container:
+Verify that the output includes a `collector-init` container:
 +
 [source,terminal]
 ----
-"containerName": "fluentd-init"
+"containerName": "collector-init"
```
Comment (periklis) [inline modules/cluster-logging-updating-logging-to-5-0.adoc:177]: I am not sure if renamed to collector anything beyond 5.4. So fluentd should still apply on 5.0 (cc @jcantrill )

File: modules/cluster-logging-updating-logging-to-5-1.adoc [hunk 5]
```diff
@@ -160,14 +160,14 @@ green  open   audit-000001
...
+$ oc get ds collector -o json | grep collector-init
 ----
 +
-Verify that the output includes a `fluentd-init` container:
+Verify that the output includes a `collector-init` container:
 +
 [source,terminal]
 ----
-"containerName": "fluentd-init"
+"containerName": "collector-init"
```
Comment (periklis) [inline modules/cluster-logging-updating-logging-to-5-1.adoc:170]: I am not sure if renamed to collector anything beyond 5.4. So fluentd should still apply on 5.1 (cc @jcantrill )

**Review by periklis — CHANGES_REQUESTED** [review]

File: logging/config/cluster-logging-systemd.adoc [hunk 6]
```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
```
Comment (jcantrill) [inline logging/config/cluster-logging-systemd.adoc:1]: I can not find where this is referenced.  It looks like a hold over from 3.11 from before CRIO when you could configure nodes to write logs to journal.  If you can not find any reference I would consider removing it.

File: modules/cluster-logging-about-collector.adoc [hunk 7]
```diff
@@ -15,7 +15,7 @@ By default, the log collector uses the following sources:
 
 If you configure the log collector to collect audit logs, it gets them from `/var/log/audit/audit.log`.
 
-The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. Fluentd collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.
+The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. collector collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.
```
Comment (jcantrill) [inline modules/cluster-logging-about-collector.adoc:18]: The 'collector'

File: modules/cluster-logging-about-components.adoc [hunk 8]
```diff
@@ -10,20 +10,21 @@ endif::[]
...
-that collects all node and container logs and writes them to a log store. You can use a centralized web UI 
+The OpenShift Logging components include a collector deployed to each node in the {product-title} cluster
+that collects all node and container logs and writes them to a log store. You can use a centralized web UI
 to create rich visualizations and dashboards with the aggregated data.
 
 The major components of OpenShift Logging are:
 
-* collection - This is the component that collects logs from the cluster, formats them, and forwards them to the log store. The current implementation is Fluentd.
-* log store - This is where the logs are stored. The default implementation is Elasticsearch. You can use the default Elasticsearch log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
+* collection - This is the component that collects logs from the cluster, formats them, and forwards them to the log store. The current implementation is the fluentd or vector collector.
```
Comment (jcantrill) [inline modules/cluster-logging-about-components.adoc:21]: implementations are...

File: modules/cluster-logging-about-components.adoc [hunk 9]
```diff
@@ -10,20 +10,21 @@ endif::[]
...
+* log store - This is where the logs are stored. The default implementation was Elasticsearch, which is now deprecated. Loki is in development as the new default store. You can use the default log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
```
Comment (jcantrill) [inline modules/cluster-logging-about-components.adoc:22]: @alanconway @periklis what do we want to say here.  loki is GA.  Elasticsearch is still the "default" implementation until probably logging CR 2.0.  Does the statement about optimization and testing still hold true for Loki

File: modules/cluster-logging-about-components.adoc [hunk 10]
```diff
@@ -10,20 +10,21 @@ endif::[]
...
+//Eng-Feedback: short-term not true for Loki? 
 * visualization - This is the UI component you can use to view logs, graphs, charts, and so forth. The current implementation is Kibana.
 
 ifndef::virt-logging[]
-This document might refer to log store or Elasticsearch, visualization or Kibana, collection or Fluentd, interchangeably, except where noted.
+This document might refer to log store or Elasticsearch, visualization or Kibana, collection or collector, interchangeably, except where noted.
```
Comment (jcantrill) [inline modules/cluster-logging-about-components.adoc:27]: collection or fluentd, vector ?

File: modules/cluster-logging-about-crd.adoc [hunk 11]
```diff
@@ -60,8 +60,8 @@ spec:
         node-role.kubernetes.io/worker: ""
   collection:
     logs:
-      type: "fluentd"
-      fluentd: {}
+      type: "collector"
```
Comment (jcantrill) [inline modules/cluster-logging-about-crd.adoc:63]: vector

File: modules/cluster-logging-about-crd.adoc [hunk 12]
```diff
@@ -110,8 +110,8 @@ spec:
       replicas: 1
   collection: <6>
     logs:
-      type: "fluentd"
-      fluentd:
+      type: "collector"
```
Comment (jcantrill) [inline modules/cluster-logging-about-crd.adoc:113]: if this type remains "fluent" we can keep the ref to resources.limits, if we use vector we should consider removing

File: modules/cluster-logging-about-logstore.adoc [hunk 13]
```diff
@@ -6,11 +6,11 @@
 [id="cluster-logging-about-logstore_{context}"]
 = About the log store
 
-By default, {product-title} uses link:https://www.elastic.co/products/elasticsearch[Elasticsearch (ES)] to store log data. Optionally, you can use the log forwarding features to forward logs to external log stores using Fluentd protocols, syslog protocols, or the {product-title} Log Forwarding API.
+By default, {product-title} uses link:https://www.elastic.co/products/elasticsearch[Elasticsearch (ES)] to store log data. Optionally, you can use the log forwarding features to forward logs to external log stores using collector protocols, syslog protocols, or the {product-title} Log Forwarding API.
```
Comment (jcantrill) [inline modules/cluster-logging-about-logstore.adoc:9]: @alanconway @periklis what to say here?

File: modules/cluster-logging-about-logstore.adoc [hunk 14]
```diff
@@ -6,11 +6,11 @@
...
+Elasticsearch organizes the log data from collector into datastores, or _indices_, then subdivides each index into multiple pieces called _shards_, which it spreads across a set of Elasticsearch nodes in an Elasticsearch cluster. You can configure Elasticsearch to make copies of the shards, called _replicas_, which Elasticsearch also spreads across the Elasticsearch nodes. The `ClusterLogging` custom resource (CR) allows you to specify how the shards are replicated to provide data redundancy and resilience to failure. You can also specify how long the different types of logs are retained using a retention policy in the `ClusterLogging` CR.
```
Comment (jcantrill) [inline modules/cluster-logging-about-logstore.adoc:13]: the collectors or collectors

File: modules/cluster-logging-about.adoc [hunk 15]
```diff
@@ -24,7 +24,7 @@ ifdef::openshift-dedicated[]
 {product-title} administrators can deploy the Red Hat OpenShift Logging Operator and the
 OpenShift Elasticsearch Operator by using the {product-title} web console and can configure logging in the
 `openshift-logging` namespace. Configuring logging will deploy Elasticsearch,
-Fluentd, and Kibana in the `openshift-logging` namespace. The operators are
+collector, and Kibana in the `openshift-logging` namespace. The operators are
```
Comment (jcantrill) [inline modules/cluster-logging-about.adoc:27]: a collector... but seems like there is additional rewrite needed to include loki

File: modules/cluster-logging-collector-alerts.adoc [hunk 16]
```diff
@@ -8,29 +8,29 @@
...
-.Fluentd Prometheus alerts
+.collector Prometheus alerts
 [cols="2,2,2,1",options="header"]
 |===
 |Alert |Message |Description |Severity
 
-|`FluentDHighErrorRate`
-|`<value> of records have resulted in an error by fluentd <instance>.`
-|The number of FluentD output errors is high, by default more than 10 in the previous 15 minutes.
+|`collectorHighErrorRate`
```
Comment (jcantrill) [inline modules/cluster-logging-collector-alerts.adoc:16]: revert.  still named this

File: modules/cluster-logging-collector-alerts.adoc [hunk 17]
```diff
@@ -8,29 +8,29 @@
...
+|`<value> of records have resulted in an error by the collector <instance>.`
+|The number of collector output errors is high, by default more than 10 in the previous 15 minutes.
 |Warning
 
-|`FluentdNodeDown`
-|`Prometheus could not scrape fluentd <instance> for more than 10m.`
-|Fluentd is reporting that Prometheus could not scrape a specific Fluentd instance.
+|`collectorNodeDown`
```
Comment (jcantrill) [inline modules/cluster-logging-collector-alerts.adoc:21]: revert

File: modules/cluster-logging-collector-alerts.adoc [hunk 18]
```diff
@@ -8,29 +8,29 @@
...
+|`Prometheus could not scrape collector <instance> for more than 10m.`
+|collector is reporting that Prometheus could not scrape a specific collector instance.
 |Critical
 
-|`FluentdQueueLengthIncreasing`
-|`In the last 12h, fluentd <instance> buffer queue length constantly increased more than 1. Current value is <value>.`
-|Fluentd is reporting that the queue size is increasing.
+|`collectorQueueLengthIncreasing`
```
Comment (jcantrill) [inline modules/cluster-logging-collector-alerts.adoc:26]: revert

File: modules/cluster-logging-collector-limits.adoc [hunk 19]
```diff
@@ -32,7 +32,7 @@ metadata:
 spec:
```
Comment (jcantrill) [inline modules/cluster-logging-collector-limits.adoc:32]: format is now:
```
collection:
  type:
  resources:
```

File: modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc [hunk 20]
```diff
@@ -27,9 +27,9 @@ Elasticsearch 6.8.4
 
 Elasticsearch 7.12.2
 
-| fluentdForward
-| fluentd forward v1
-a| fluentd 1.7.4
+| collectorForward
```
Comment (jcantrill) [inline modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc:30]: revert

File: modules/cluster-logging-collector-tolerations.adoc [hunk 21]
```diff
@@ -53,8 +53,8 @@ that do not match.
 ----
   collection:
     logs:
-      type: "fluentd"
```
Comment (jcantrill) [inline modules/cluster-logging-collector-tolerations.adoc:56]: format changes to 
```
collection:
  type:
  tolerations
```

File: modules/cluster-logging-collector-tuning.adoc [hunk 22]
```diff
@@ -1,169 +0,0 @@
-// Module included in the following assemblies:
```
Comment (jcantrill) [inline modules/cluster-logging-collector-tuning.adoc:1]: this is all relevent to fluentd

File: modules/cluster-logging-configuring-image-about.adoc [hunk 23]
```diff
@@ -20,9 +20,9 @@ $ oc -n openshift-logging set env deployment/cluster-logging-operator --list | g
 .Example output
 [source,terminal]
 ----
-FLUENTD_IMAGE=registry.redhat.io/openshift-logging/fluentd-rhel8:latest@sha256:ffdf79da7386871d2bc24cd937e02284b30f85a9979dc8c635fb73021cbca2f3 <1>
+collector_IMAGE=registry.redhat.io/openshift-logging/collector-rhel8:latest@sha256:ffdf79da7386871d2bc24cd937e02284b30f85a9979dc8c635fb73021cbca2f3 <1>
```
Comment (jcantrill) [inline modules/cluster-logging-configuring-image-about.adoc:23]: ther3e are multiple images here https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Ajcantrill%2Fcluster-logging-operator.git&branch=log2759_val&file=bundle%2Fmanifests%2Fclusterlogging.clusterserviceversion.yaml&editor=JetBrains&version=v1.2.2&start_row=413&start_col=25&end_row=413&end_col=25

File: modules/cluster-logging-configuring-node-selector.adoc [hunk 24]
```diff
@@ -49,12 +49,12 @@ spec:
       replicas: 1
   collection:
     logs:
-      type: "fluentd"
-      fluentd:
+      type: "collector"
```
Comment (jcantrill) [inline modules/cluster-logging-configuring-node-selector.adoc:52]:
```
collection:
  type:
  nodeSelector:

File: modules/cluster-logging-cpu-memory.adoc [hunk 25]
```diff
@@ -62,8 +62,8 @@ spec:
       replicas: 2
   collection:
     logs:
-      type: "fluentd"
```
Comment (jcantrill) [inline modules/cluster-logging-cpu-memory.adoc:65]: see previous ref

File: modules/cluster-logging-deploy-cli.adoc [hunk 26]
```diff
@@ -325,8 +325,8 @@ spec:
         node-role.kubernetes.io/worker: ""
   collection:
     logs:
```
Comment (jcantrill) [inline modules/cluster-logging-deploy-cli.adoc:327]: api changed

File: modules/cluster-logging-deploy-console.adoc [hunk 27]
```diff
@@ -167,8 +167,8 @@ spec:
         node-role.kubernetes.io/worker: ""
   collection:
     logs:
```
Comment (jcantrill) [inline modules/cluster-logging-deploy-console.adoc:169]: api change

File: modules/cluster-logging-deploy-storage-considerations.adoc [hunk 28]
```diff
@@ -63,7 +63,7 @@ Calculating the total logging throughput and disk space required for your {produ
 ----
 ////
 
-Fluentd ships any logs from *systemd journal* and */var/log/containers/* to Elasticsearch.
+collector ships any logs from *systemd journal* and */var/log/containers/* to Elasticsearch.
```
Comment (jcantrill) [inline modules/cluster-logging-deploy-storage-considerations.adoc:66]: /var/log/pods/*

File: modules/cluster-logging-deploying-about.adoc [hunk 29]
```diff
@@ -43,15 +43,15 @@ spec:
       type: "elasticsearch"
   collection:
     logs:
```
Comment (jcantrill) [inline modules/cluster-logging-deploying-about.adoc:45]: api change

**Review by jcantrill — CHANGES_REQUESTED** [review]

File: logging/cluster-logging-exported-fields.adoc [hunk 30]
```diff
@@ -10,7 +10,7 @@ The following fields can be present in log records exported by OpenShift Logging
 
 To search these fields from Elasticsearch and Kibana, use the full dotted field name when searching. For example, with an Elasticsearch */_search URL*, to look for a Kubernetes pod name, use `/_search/q=kubernetes.pod_name:name-of-my-pod`.
```
Comment (anpingli) [inline logging/cluster-logging-exported-fields.adoc:11]: _search?q=kubernetes.pod_name:name-of-my-pod`

File: modules/cluster-logging-about-crd.adoc (hunk 11 above)
Comment (anpingli) [inline modules/cluster-logging-about-crd.adoc:63]: Type should be either fluentd or vector.

File: modules/cluster-logging-about-crd.adoc [hunk 31]
```diff
@@ -60,8 +60,8 @@ spec:
         node-role.kubernetes.io/worker: ""
   collection:
     logs:
-      type: "fluentd"
-      fluentd: {}
+      type: "collector"
+      collector: {}
```
Comment (anpingli) [inline modules/cluster-logging-about-crd.adoc:64]: There isn't properties collector.
https://github.com/openshift/cluster-logging-operator/blob/release-5.5/bundle/manifests/logging.openshift.io_clusterloggings.yaml#L51

File: modules/cluster-logging-about-crd.adoc [hunk 32]
```diff
@@ -110,8 +110,8 @@ spec:
       replicas: 1
   collection: <6>
     logs:
-      type: "fluentd"
-      fluentd:
+      type: "collector"
+      collector:
```
Comment (anpingli) [inline modules/cluster-logging-about-crd.adoc:114]: Same as above

File: modules/cluster-logging-clo-status.adoc [hunk 33]
```diff
@@ -41,25 +41,25 @@ kind: ClusterLogging
 status:  <1>
   collection:
     logs:
-      fluentdStatus:
-        daemonSet: fluentd  <2>
+      collectorStatus:
```
Comment (anpingli) [inline modules/cluster-logging-clo-status.adoc:44]: It is fluentdStatus when use fluentd as collector.

Comment (anpingli) [reply modules/cluster-logging-clo-status.adoc:44]: Filed a ticket https://issues.redhat.com/browse/LOG-3126

File: modules/cluster-logging-clo-status.adoc [hunk 34]
```diff
@@ -215,16 +215,16 @@ A status message similar to the following indicates that the requested PVC could
...
 .Example output
 [source,yaml]
 ----
 Status:
   Collection:
     Logs:
-      Fluentd Status:
-        Daemon Set:  fluentd
+      collector Status:
+        Daemon Set:  collector
```
Comment (anpingli) [inline modules/cluster-logging-clo-status.adoc:227]: https://issues.redhat.com/browse/LOG-3126

File: modules/cluster-logging-collector-alerts.adoc (hunk 16 above)
Comment (anpingli) [inline modules/cluster-logging-collector-alerts.adoc:16]: The alert names are still Fluent-xxxx. Filed a bug https://issues.redhat.com/browse/LOG-3127

File: modules/cluster-logging-collector-alerts.adoc (hunk 17 above)
Comment (anpingli) [inline modules/cluster-logging-collector-alerts.adoc:21]: Same as above

File: modules/cluster-logging-collector-limits.adoc [hunk 35]
```diff
@@ -32,7 +32,7 @@ metadata:
...
   collection:
     logs:
-      fluentd:
+      collector:
```
Comment (anpingli) [inline modules/cluster-logging-collector-limits.adoc:35]: collector is not a  valid  properties. @jcantrill  Do we need a bug?
https://github.com/openshift/cluster-logging-operator/blob/master/bundle/manifests/logging.openshift.io_clusterloggings.yaml#L54

File: modules/cluster-logging-collector-log-forward-about.adoc [hunk 36]
```diff
@@ -17,7 +17,7 @@ An _output_ is the destination for log data and a pipeline defines simple routin
 An output can be either:
```
Comment (anpingli) [inline modules/cluster-logging-collector-log-forward-about.adoc:17]: LogForwarding API had been removed. I think we can drop this file and replace with ClusterLogForwarder

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 37]
```diff
@@ -9,7 +9,7 @@ Forwarding cluster logs to external third-party systems requires a combination o
 --
 * `elasticsearch`. An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
 
-* `fluentdForward`. An external log aggregation solution that supports Fluentd. This option uses the Fluentd *forward* protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a *shared_key* field in a secret. Shared-key authentication can be used with or without TLS.
+* `collectorForward`. An external log aggregation solution that supports collector. This option uses the collector *forward* protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a *shared_key* field in a secret. Shared-key authentication can be used with or without TLS.
```
Comment (anpingli) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:12]: fluentdForward is fluentd only feature. Please roll back the changes

File: modules/cluster-logging-collector-log-forwarding-supported-plugins-5-1.adoc [hunk 38]
```diff
@@ -22,9 +22,9 @@ Elasticsearch 6.8.4
 
 Elasticsearch 7.12.2
 
-| fluentdForward
-| fluentd forward v1
-a| fluentd 1.7.4
+| collectorForward
```
Comment (anpingli) [inline modules/cluster-logging-collector-log-forwarding-supported-plugins-5-1.adoc:25]: collectorForward -> fluentdForward

File: modules/cluster-logging-collector-log-forwarding-supported-plugins-5-1.adoc [hunk 39]
```diff
@@ -22,9 +22,9 @@ Elasticsearch 6.8.4
...
+| collector forward v1
+a| collector 1.7.4
```
Comment (anpingli) [inline modules/cluster-logging-collector-log-forwarding-supported-plugins-5-1.adoc:27]: collector-> fluentd

File: modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc (hunk 20 above)
Comment (anpingli) [inline modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc:30]: collectorForward->fluentdForward

File: modules/cluster-logging-collector-tolerations.adoc [hunk 40]
```diff
@@ -53,8 +53,8 @@ that do not match.
...
-      fluentd:
+      type: "collector"
```
Comment (anpingli) [inline modules/cluster-logging-collector-tolerations.adoc:56]: type should be either fluentd or vector, it can not be 'collector'

**Comment by anpingli** [comment]
@libander  Lots of changes from fluentd-> collector are wrong. 
Please refer to CRD below to correct them.

https://github.com/openshift/cluster-logging-operator/blob/release-5.5/bundle/manifests/logging.openshift.io_clusterloggings.yaml
https://github.com/openshift/cluster-logging-operator/blob/release-5.5/bundle/manifests/logging.openshift.io_clusterlogforwarders.yaml

File: modules/cluster-logging-configuring-image-about.adoc (hunk 23 above)
Comment (libander) [reply modules/cluster-logging-configuring-image-about.adoc:23]: reverted, text already states your list may differ based on config.

File: logging/cluster-logging-eventrouter.adoc [hunk 41]
```diff
@@ -8,11 +8,11 @@ toc::[]
 
 The {product-title} Event Router is a pod that watches Kubernetes events and logs them for collection by OpenShift Logging. You must manually deploy the Event Router.
 
-The Event Router collects events from all projects and writes them to `STDOUT`. Fluentd collects those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index.
+The Event Router collects events from all projects and writes them to `STDOUT`. The The collector collect those events and forwards them into the {product-title} log store, writing these events to the `infra` index.
```
Comment (jc-berger) [inline logging/cluster-logging-eventrouter.adoc:11]: Double `the`

`The collector collects those events and forwards them into...`

File: modules/cluster-logging-about-collector.adoc [hunk 42]
```diff
@@ -15,7 +15,7 @@ By default, the log collector uses the following sources:
 
 If you configure the log collector to collect audit logs, it gets them from `/var/log/audit/audit.log`.
 
-The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. Fluentd collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.
+The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. The collector collect the logs from these sources and forwards them internally or externally as you configure in {product-title}.
```
Comment (jc-berger) [inline modules/cluster-logging-about-collector.adoc:18]: Verb tense 

`The collector collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.`

File: modules/cluster-logging-eventrouter-about.adoc [hunk 43]
```diff
@@ -7,6 +7,6 @@
 = About event routing
 
 The Event Router is a pod that watches {product-title} events so they can be collected by OpenShift Logging.
-The Event Router collects events from all projects and writes them to `STDOUT`. Fluentd collects those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index. 
+The Event Router collects events from all projects and writes them to `STDOUT`. The collector collect those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index.
```
Comment (jc-berger) [inline modules/cluster-logging-eventrouter-about.adoc:10]: Verb tense

`The collector collects those events and forwards them into the {product-title} Elasticsearch instance. Elasticsearch indexes the events to the `infra` index.`

**Review by jc-berger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**